src/i18n/DE/grid-resources.ts

```diff
@@ -2,128 +2,128 @@ import { IGridResourceStrings } from 'igniteui-angular';
-  igx_grid_filter_row_boolean_placeholder: 'All',
-  igx_grid_filter_row_date_placeholder: 'Pick up date',
-  igx_grid_filter_operator_and: 'And',
-  igx_grid_filter_operator_or: 'Or',
-  igx_grid_complex_filter: 'Complex Filter',
-  igx_grid_filter_contains: 'Contains',
-  igx_grid_filter_doesNotContain: 'Does Not Contain',
-  igx_grid_filter_startsWith: 'Starts With',
-  igx_grid_filter_endsWith: 'Ends With',
-  igx_grid_filter_equals: 'Equals',
-  igx_grid_filter_doesNotEqual: 'Does Not Equal',
-  igx_grid_filter_empty: 'Empty',
-  igx_grid_filter_notEmpty: 'Not Empty',
+  igx_grid_filter_row_close: 'Schließen',
+  igx_grid_filter_row_reset: 'Zurücksetzen',
+  igx_grid_filter_row_placeholder: 'Füge Filterwert hinzu',
+  igx_grid_filter_row_boolean_placeholder: 'Alle',
+  igx_grid_filter_row_date_placeholder: 'Wähle Datum',
+  igx_grid_filter_operator_and: 'Und',
+  igx_grid_filter_operator_or: 'Oder',
+  igx_grid_complex_filter: 'Komplexer Filter',
+  igx_grid_filter_contains: 'Enthält',
+  igx_grid_filter_doesNotContain: 'Enthält nicht',
+  igx_grid_filter_startsWith: 'Beginnt mit',
+  igx_grid_filter_endsWith: 'Endet mit',
+  igx_grid_filter_equals: 'Gleich',
+  igx_grid_filter_doesNotEqual: 'Nicht gleich',
+  igx_grid_filter_empty: 'Leer',
+  igx_grid_filter_notEmpty: 'Nicht leer',
   igx_grid_filter_null: 'Null',
-  igx_grid_filter_notNull: 'Not Null',
-  igx_grid_filter_before: 'Before',
-  igx_grid_filter_after: 'After',
-  igx_grid_filter_today: 'Today',
-  igx_grid_filter_yesterday: 'Yesterday',
-  igx_grid_filter_thisMonth: 'This Month',
-  igx_grid_filter_lastMonth: 'Last Month',
-  igx_grid_filter_nextMonth: 'Next Month',
-  igx_grid_filter_thisYear: 'This Year',
-  igx_grid_filter_lastYear: 'Last Year',
-  igx_grid_filter_nextYear: 'Next Year',
-  igx_grid_filter_greaterThan: 'Greater Than',
-  igx_grid_filter_lessThan: 'Less Than',
-  igx_grid_filter_greaterThanOrEqualTo: 'Greater Than Or Equal To',
-  igx_grid_filter_lessThanOrEqualTo: 'Less Than Or Equal To',
-  igx_grid_filter_true: 'True',
-  igx_grid_filter_false: 'False',
-  igx_grid_filter_all: 'All',
-  igx_grid_filter_condition_placeholder: 'Select filter',
-  igx_grid_summary_count: 'Count',
+  igx_grid_filter_notNull: 'Nicht Null',
+  igx_grid_filter_before: 'Vor',
+  igx_grid_filter_after: 'Nach',
+  igx_grid_filter_today: 'Heute',
+  igx_grid_filter_yesterday: 'Gestern',
+  igx_grid_filter_thisMonth: 'Diesen Monat',
+  igx_grid_filter_lastMonth: 'Letzten Monat',
+  igx_grid_filter_nextMonth: 'Nächsten Monat',
+  igx_grid_filter_thisYear: 'Dieses Jahr',
+  igx_grid_filter_lastYear: 'Letztes Jahr',
+  igx_grid_filter_nextYear: 'Nächstes Jahr',
+  igx_grid_filter_greaterThan: 'Größer als',
+  igx_grid_filter_lessThan: 'Kleiner als',
+  igx_grid_filter_greaterThanOrEqualTo: 'Größer als oder gleich',
+  igx_grid_filter_lessThanOrEqualTo: 'Kleiner als oder gleich',
+  igx_grid_filter_true: 'Wahr',
+  igx_grid_filter_false: 'Falsche',
```

Comment:
If this is supposed to be a boolean filter (like in https://www.igniteui.com/grid/custom-conditions-filtering), e.g. X is true or bool is false, then it should be "Falsch" as the translation.

Reply:
Oops. Typo